apiserver/common/permissions.go

```diff
@@ -100,8 +103,14 @@ func HasModelAdmin(
 	modelTag names.ModelTag,
 ) (bool, error) {
 	// superusers have admin for all models.
-	if isSuperUser, err := authorizer.HasPermission(permission.SuperuserAccess, controllerTag); err != nil || isSuperUser {
+	isSuperUser, err := authorizer.HasPermission(permission.SuperuserAccess, controllerTag)
```

Comment:
isSuperuser return arg not needed since it will error if no perm

Reply:
Done & Done